server.go

```diff
@@ -542,15 +542,15 @@ func (srv *Server) serve(w *response, wg *sync.WaitGroup) {
 	wg.Done()
 }
 
-func (srv *Server) disposeBuffer(w *response) {
-	if w.udp != nil && cap(w.msg) == srv.UDPSize {
-		srv.udpPool.Put(w.msg[:srv.UDPSize])
+func (srv *Server) disposeBuffer(udp bool, m []byte) {
+	if udp && cap(m) == srv.UDPSize {
+		srv.udpPool.Put(m[:srv.UDPSize])
 	}
-	w.msg = nil
+	m = nil
```

Comment:
This isn’t needed anymore. It won’t block GC of the buffer as it’s only acessable during this function call.

Reply:
the `= nil` part you mean?